Plugins/Public/event/Main.cpp

```diff
@@ -556,16 +583,16 @@ void LoadSettings()
 	ConPrint(L"EVENT DEBUG: Loaded %u event player data\n", iLoaded3);
 
 	LoadIDs();
```

Comment:
It might be a bit much, but I think it would be neat if you could have a validation of a trade event. Namely, calling the following within a try-catch block, loading it in if it succeeds and printing error information on the screen via ConPrint and/or AddLog()

```
Universe::IBase* base = Universe::get_base()
const Universe::ISystem* sys = Universe::get_system()
const GoodInfo* gi = GoodList::find_by_id()
```